js/apps/admin-ui/src/realm-settings/TokensTab.tsx

```diff
@@ -85,6 +88,30 @@ export const RealmSettingsTokensTab = ({
     defaultValue: false,
   });
 
+  const signedMetadataEnabled =
+    useWatch({
+      control,
+      name: convertAttributeNameToForm(
+        "attributes.oid4vci.signed_metadata.enabled",
+      ),
+      defaultValue: realm.attributes?.["oid4vci.signed_metadata.enabled"],
+    }) === "true";
+
+  const encryptionRequired =
+    useWatch({
+      control,
+      name: convertAttributeNameToForm(
+        "attributes.oid4vci.encryption.required",
+      ),
+      defaultValue: realm.attributes?.["oid4vci.encryption.required"],
+    }) === "true";
```

Comment:
For consistency with the rest of the codebase, prefer doing so where the attribute is used.
```suggestion
  const signedMetadataEnabled =
    useWatch({
      control,
      name: convertAttributeNameToForm(
        "attributes.oid4vci.signed_metadata.enabled",
      ),
      defaultValue: realm.attributes?.["oid4vci.signed_metadata.enabled"],
    });

  const encryptionRequired =
    useWatch({
      control,
      name: convertAttributeNameToForm(
        "attributes.oid4vci.encryption.required",
      ),
      defaultValue: realm.attributes?.["oid4vci.encryption.required"],
    });
```